examples/Example/src/new_api/drag_n_drop/Draggable.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent } from 'react';
+import { StyleSheet } from 'react-native';
+import {
+  PanGestureHandlerEventPayload,
+  Gesture,
+  GestureDetector,
+} from 'react-native-gesture-handler';
+import Animated, { useAnimatedStyle } from 'react-native-reanimated';
+import { TapGesture } from '../../../../../src/handlers/gestures/tapGesture';
```

Comment:
Should be imported from RNGH.

Reply:
It's not exported so need to be fixed.

Reply:
@j-piasecki

Reply:
Fixed in https://github.com/software-mansion/react-native-gesture-handler/pull/1641

Reply:
Since the fix is merged now, could you update the example @enterteg?